.github/scripts/update-release-notes.js

```diff
@@ -0,0 +1,165 @@
+                        //## Aug 8, 2023
+                        console.error(`## ${shortMonth} ${day}, ${year}`);
+                        //### Android BOM 2.3.1
+                        console.error(`### Android BOM  ${version}`);
+                        console.error(releaseNotes)
+
+                        const releaseMdPath = "./src/pages/documentation/release-notes/index.md";
+
+                        const releaseNotesHeader = "# Release notes";
+                        // Read the contents of the markdown file.
+                        const contentArray = fs.readFileSync(releaseMdPath, "utf8").toString().split("\n");
+                         // Find the index of the release notes header.
+                        const releaseNotesHeaderIndex = contentArray.indexOf(releaseNotesHeader);
+                        console.error(releaseNotesHeaderIndex)
+
+                        if (releaseFileContainsLineStartWith(`### Android BOM  ${version}`) == true) {
+                            console.error("already updated")
+                            return
+                        }
+                        contentArray.splice(releaseNotesHeaderIndex + 1,0,"",`## ${shortMonth} ${day}, ${year}`,"",`### Android BOM  ${version}`, "",updateBOMReleaseNotesForAdobeIO(releaseNotes));
+                        fs.writeFile(releaseMdPath, contentArray.join("\n"), function (err) {
+                            if (err) return console.log(err);
+                          });
+                    })
+                }
+                
+            }
+        });
+    });
+ 
+});
+ 
+reqGet.end();
+reqGet.on('error', function(e) {
+    console.error(e);
+});
+
+}
+
+function updateBOMReleaseNotesForAdobeIO(releaseNotesString) {
+    releaseNotesAsArray = releaseNotesString.split("\n")
+    releaseNotesForAdobeIO = []
+    releaseNotesForAdobeIO.push("* This BOM ([Bill of Materials](https://central.sonatype.com/artifact/com.adobe.marketing.mobile/sdk-bom)) release includes changes to the following Android extensions.")
+    releaseNotesForAdobeIO.push("")
+    releaseNotesForAdobeIO.push("<Accordion>")
+    releaseNotesForAdobeIO.push("")
+    releaseNotesForAdobeIO.push("<AccordionItem header='Expand'>")
+    releaseNotesForAdobeIO.push("")
+    for(line in releaseNotesAsArray) {
+        if (releaseNotesAsArray[line].trim().startsWith("|")) {
+            releaseNotesForAdobeIO.push(releaseNotesAsArray[line])
+        }
+    }
+    releaseNotesForAdobeIO.push("")
+    releaseNotesForAdobeIO.push("</AccordionItem>")
+    releaseNotesForAdobeIO.push("")
+    releaseNotesForAdobeIO.push("</Accordion>")
+    return releaseNotesForAdobeIO.join("\n")
+}
+
+function releaseFileContainsLineStartWith(lineStartWithString){
+    try {
+        execSync(`grep -E "^${lineStartWithString}" ./src/pages/documentation/release-notes/index.md`, { stdio: 'ignore' });
+        return true;
+      } catch (e) {
+        return false;
+      }
+}
+
+updateBOMReleaseNotes()
```

Comment:
I would prefer automating this process for all of our repositories. We can maintain a list of repos and header names, then iterate through them, adding the contents of most recent GitHub release to our release notes that way teams only have to update their GH release with release notes (Android-core would require additional logic as it hosts 4 extensions)

Reply:
I was planning to do it in a separate PR. Before that, I'd prefer to discuss this solution in the SDK meeting and make sure all Android extension repos can follow the convention.

Reply:
Let us discuss it internally and try to implement this for all our Android and iOS repos.

Reply:
Agree with Praveen! In future, it would be great to have the release notes updated here using content from GitHub releases.